index.d.ts

```diff
@@ -24,22 +25,40 @@ declare module 'plaid' {
     offset?: number;
   }
 
+  interface AssetReportUser {
```

Comment:
The `AssetReportUser` fields are all optional on the create request, so let's make the types `client_user_id?: string`, etc.

Reply:
⚡️

Reply:
is `client_user_id: null` allowed @joyzheng @dsfish ? (similar question for the other optional fields... we should be clear about this in the typings).

Reply:
⚡️